.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
```

Comment:
no need to include this file

Reply:
removed it again - just thought it would be nice to enforce the line length limit :blush: